.github/workflows/ci.yml

```
@@ -370,18 +370,18 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: GraphQL Inspector (V1)
-        uses: kamilkisiela/graphql-inspector@master
+        uses: kamilkisiela/graphql-inspector@main
```

According to https://github.com/kamilkisiela/graphql-inspector they haven't renamed their main branch

```suggestion
        uses: kamilkisiela/graphql-inspector@master
```